src/implementation/Server/HTTPServer/HTTPServer.ts

```diff
@@ -17,7 +18,7 @@ export default class HTTPServer implements IServer {
   server: IServerType;
   serverAddress: string;
   serverImpl: IServerImplType;
-  serverStartupDelay = 1000; // @todo: use health api https://docs.dapr.io/reference/api/health_api/
+  serverStartupDelay = 1000;
```

Comment:
Same comment here about the TODO. It might also be worth changing the variable name to `serverStartupDelayMs` to clearly indicate that this is milliseconds.